build-support/bin/release.sh

```diff
@@ -469,7 +469,7 @@ function fetch_and_check_prebuilt_wheels() {
   do
     packages=($(find_pkg "${PACKAGE}" "${PANTS_UNSTABLE_VERSION}" "${check_dir}"))
     if [ ${#packages[@]} -eq 0 ]; then
-      missing+=("${NAME}")
+      missing+=("${PACKAGE}")
```

Comment:
Is this change breaking anything for you at the moment, or you only noticed it and are being a good citizen? I can roll it into https://github.com/pantsbuild/pants/pull/7197 if you prefer Stu to keep this PR more focused.

Reply:
It's not breaking anything at the moment, just needed to be cleaned up.

Reply:
Sounds good! I'll add it to my PR that makes a bunch of `releases.sh` changes already, meaning it can be removed here and the PR title can be shortened.

Reply:
So looking more into it, there are 5 distinct places we use `${NAME}` in `release.sh`. I'm not entirely sure where it gets defined, but it seems like they're getting resolved as we run these functions when doing a manual release, and no one has seemed to notice.

I'm going to leave it out of my PR after all. If we still want to address it, I think this change would be best broken out into a dedicated PR.